tests/ui/many_single_char_names.rs

```diff
@@ -1,6 +1,7 @@
 #[warn(clippy::many_single_char_names)]
 
 fn bla() {
+    let x: i32;
```

Comment:
I'd rather lower the default threshold by one, than modifying the test. 

https://github.com/rust-lang/rust-clippy/blob/c496f4e63f5621872060b21793c5ec4ddf0e4a35/clippy_lints/src/utils/conf.rs#L137-L138

(the documentation of this config param is now more correct than before 😄)